documentation/jetty-documentation/src/main/asciidoc/programming-guide/server/sessions/session-sessionhandler.adoc

```diff
@@ -14,115 +14,113 @@
 [[pg-server-session-handler]]
 ==== The SessionHandler
 
-Each context can have a single `SessionHandler`.
-The purpose of the `SessionHandler` is to interact with the `Request` and `Response` to create, maintain and propagate sessions.
-It also calls the context-level session listeners at appropriate points in the session lifecycle.
+A `SessionHandler` is a `Handler` that implements the `SessionManager`, and is thus responsible for the creation, maintenance and propagation of sessions.
+There are `SessionHandlers` for both the core and the various servlet environments.
+
+Note that in the servlet environments, each `ServletContextHandler` or `WebAppContext` has at most a single `SessionHandler`.
 
 ===== Configuration
 
-The majority of configuration for the link:{javadoc-url}/org/eclipse/jetty/server/session/SessionHandler.html[SessionHandler] can be done via `web.xml` `<session-config>` declarations, or the `javax.servlet.SessionCookieConfig` api.
-There are also a few jetty-specific configuration options that we will cover here:
+The majority of configuration for the `SessionHandler` can be done via `web.xml` `<session-config>` declarations, or the `javax.servlet.SessionCookieConfig` api.
+There are also a few jetty-specific configuration methods that we will cover here:
```

Comment:
```suggestion
There are also a few Jetty-specific configuration methods:
```

Reply:
That text was from an older version and no longer exists.